platform/core/src/compatibility/dialogPolyfill.js

```diff
@@ -1,458 +0,0 @@
-var dialogPolyfill = (function() {
```

Comment:
This would polyfill dialog on IE 11 -- I'm not sure if we're using it anywhere or not.
Quickest way to verify would be trying to use CINE dialog in IE11 with this removed?

Reply:
I looked for the <dialog> tag in the code but didn't see it anywhere anymore